pokemon/convert.py

```diff
@@ -66,23 +57,21 @@ def convert_image_to_ascii(image, new_width):
     len_pixels_to_chars = len(pixels_to_chars)
 
     image_ascii = [pixels_to_chars[index: index + new_width*2] for index in
-            xrange(0, len_pixels_to_chars, new_width*2)]
-
+            range(0, len_pixels_to_chars, new_width*2)]
     return "\n".join(image_ascii)
 
-def handle_image_conversion(image_filepath,new_width=100):
+def handle_image_conversion(image_filepath,new_width):
```

Comment:
Where is this used with the default?

Reply:
Ah, once again I thought this was something I added when trying to work through an issue back in December and took it out. Will add it back in.